packages/scripts/README.md

```diff
@@ -207,8 +213,59 @@ _Example:_
 ```
 
 This is how you execute those scripts using the presented setup:
+
 * `npm run test:unit` - runs all unit tests.
 * `npm run test:unit:help` - prints all available options to configure unit tests runner.
 * `npm run test:unit:watch` - runs all unit tests in the watch mode.
 
+## Webpack config
+
+The `build` and `start` commands use [Webpack](https://webpack.js.org/) behind the scenes. Webpack is a tool that helps you transform your code into something else. For example: it can take code written in ESNext and output ES5 compatible code that is minified for production.
+
+### Default Webpack config
+
+`@wordpress/scripts` bundles the default Webpack config used as a base by the WordPress editor. These are the defaults:
+
+* [Entry](https://webpack.js.org/configuration/entry-context/#entry): `src/index.js`
+* [Output](https://webpack.js.org/configuration/output): `build/index.js`
+* [Externals](https://webpack.js.org/configuration/externals). These are the transformations done to imports:
+
+Package | Input syntax | Output
+--- | --- | ---
+React | `import x from React;` | `var x = this.wp.react.x;`
+ReactDOM | `import x from ReactDOM;` | `var x = this.wp.react-dom.x;`
+moment | `import x from moment;` | `var x = this.wp.moment.x;`
+jQuery | `import x from jQuery;` | `var x = this.wp.jQuery.x;`
+lodash | `import x from lodash;` | `var x = this.wp.lodash.x;`
+lodash-es | `import x from lodash-es;` | `var x = this.wp.lodash-es.x;`
+Any WordPress package | `import x from '@wordpress/packageName` | `var x = this.wp.package-name.x`
+
+### Provide your own Webpack config
+
+Should there be any situation where you want to provide your own Webpack config, you can do so. The `build` and `start` commands will use your provided file when:
+
+* the command receives a `--config` argument. Example: `wp-scripts build --config my-own-webpack-config.js`.
+* there is a file called `webpack.config.js` or `webpack.config.babel.js` in the top-level directory of your package (at the same level than your `package.json`).
+
+### Extend the default config
```

Comment:
I'm still not sure if this is a good idea to promote this technique. I believe this was also what @youknowriad referred to when he raised the question of whether we want to expose webpack config as its own package. The biggest issue we both see here is that it will build the expectation that we are going to help to solve the various issues which people might have when trying to create their own config.

Reply:
060e1fb Happy to remove it now and bring it back later if/when people ask for this.